internal/primitive/query.go

```diff
@@ -51,7 +60,15 @@ func (e *Query) InitializeDB(ctx context.Context, i modconfig.Input) (*sql.DB, e
 	}
 
 	dbConnectionString := i[schema.AttributeTypeConnectionString].(string)
-	db, err = sql.Open("postgres", dbConnectionString)
+	switch {
+	case strings.Contains(dbConnectionString, "postgres://"):
+		db, err = sql.Open(DriverPostgres, dbConnectionString)
+	case strings.Contains(dbConnectionString, "@tcp("): // Common pattern in MySQL DSN
```

Comment:
Is this right? I'm reading this: https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html

And it looks like the prefix should be `mysql` or `mysqlx`.

Where did you get this `@tcp` from?